modules/core/src/core/animation-loop-proxy.js

```diff
@@ -157,6 +164,12 @@ export default class AnimationLoopProxy {
     return this;
   }
 
+  firstFrame() {
+    return new Promise(resolve => {
```

Comment:
A potential issue here is if `firstFrame` is called several times.

By also saving the promise you can handle this.

```
if (!this._firstFramePromise) {
  ...
}
return this._firstFramePromise) {
}
```

Since it can be called any time, maybe `nextFrame()` could be a better name.

Since we are not following the `verbNoun` pattern we could just make it into a getter and document that it evaluates to a promise that will resolve on the next frame.

Reply:
Good point re: `firstFrame` getting called multiple times. We wouldn't want to "unsubscribe" any callbacks added to the first promise. Will fix.

Re: `nextFrame` - as it is currently implemented, you can call `firstFrame()` whenever you want, but the promise it returns will only resolve on the first frame _after a `start()`_. We _could_ change the functionality so that it returns a promise that is always resolved on the next frame, though. (Isn't this basically the functionality of `requestAnimationFrame` though?)